packages/@aws-cdk/cdk/lib/stack.ts

```diff
@@ -94,20 +95,46 @@ export class Stack extends Construct implements ITaggable {
   public readonly stackName: string;
 
   /**
-   * The region into which this stack will be deployed.
+   * The AWS region into which this stack will be deployed (e.g. `us-west-2`).
    *
-   * This will be a concrete value only if an account was specified in `env`
-   * when the stack was defined. Otherwise, it will be a string that resolves to
-   * `{ "Ref": "AWS::Region" }`
+   * This value is resolved according to the following rules:
+   *
+   * 1. The value provided to `env.region` when the stack is defined. This can
+   *    either be a concerete region (e.g. `us-west-2`) or the `Aws.region`
```

Comment:
Does this mean I can test if this is a token using `stack.region === Aws.region`? Because this documentation somewhat implies this to me...

Reply:
You can. `Aws.region` is a constant, you totally can.